namespaces/docker.yml

```diff
@@ -9,3 +9,103 @@ namespace:
     type: string
     description: >
       Docker container id
+  - name: event
+    type: group
+    description: >
+      Event emitted by docker daemon that describes some activity that occurred in the past
+    fields:
+    - name: action
+      type: string
+      example: create
+      description: >
+        What activity occurred
+    - name: status
+      type: string
+      example: create
+      description: >
+        What is the result of the activity
+    - name: type
+      type: string
+      example: container
+      description: >
+        Container or image or volume or network or daemon or plugin or service or node or secret or config
+    - name: time
+      type: date
+      example: 2017-08-22T14:47:29.146379+00:00
+      description: >
+        Date of the occurrence of the event
```

Comment:
@josefkarasek, thanks, so perhaps updating the description to say it is a timestamp would be worth the effort.